src/components/DataSelector/VariablesSelect/index.tsx

```diff
@@ -1,13 +1,13 @@
 import { Alert, AlertDescription, AlertIcon, AlertTitle } from "@chakra-ui/alert";
 
-import CubeVariablesSelect from "./CubeVariablesSelect";
+import RendersOptionSelect from "./RendersOptionSelect";
 import { SelectProps } from "./types";
 
 function VariablesSelect({ collection, addLayer }: SelectProps) {
-  const cubeVariables = collection.stac["cube:variables"];
+  const renderOptions = collection.stac["cube:variables"] || collection.stac.renders;
 
-  if (cubeVariables) {
-    return <CubeVariablesSelect collection={collection} addLayer={addLayer} />
+  if (renderOptions) {
+    return <RendersOptionSelect collection={collection} addLayer={addLayer} />
```

Comment:
I'm wondering if we still need the subcomponent here? The logic in `RenderOptionsSelect` now seems to be generic enough so it handles data cubes, HLS and plain STAC collections with the render extension so I don't see the need for the extra component here.

Reply:
🤔 that makes sense to me, I'll try it that way. I also realized this is still under VariablesSelect so I think I will change that to RendersOptionSelect